doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml

```diff
@@ -286,8 +286,13 @@ The following console application creates updates data within the **AdventureWor
 			<exception cref="T:System.InvalidOperationException">
 				The
 				<see cref="T:Microsoft.Data.SqlClient.SqlConnection" />
-				closed or dropped during a streaming operation. For more information about streaming, see [SqlClient Streaming Support](/sql/connect/ado-net/sqlclient-streaming-support).
-			</exception>
+        closed or dropped during a streaming operation. For more information about streaming, see [SqlClient Streaming Support](/sql/connect/ado-net/sqlclient-streaming-support).
+
+        - or -
+
+        <see cref="P:Microssoft.Data.SqlClient.SqlCommand.EnableOptimizedParameterBinding" />
+        is set to true and a parameter with direction Output or InputOutput has been added to the <see cref="P:Microsoft.Data.SqlClient.SqlCommand.Parameters" />Parameters collection.
```

Comment:
Just remove the extra "**Parameters**" word at the end of the sentence.
- Applies to the other sections too.

`is set to true and a parameter with direction Output or InputOutput has been added to the <see cref="P:Microsoft.Data.SqlClient.SqlCommand.Parameters" /> collection.`

Reply:
Done. 
The enclave tests seem to be failing for some reason.

Reply:
You should add `ConditionalFact` over your tests. Because those agents don't set up `TCPConnectionString`.
I suggest preparing a set of connection strings to feed up your tests or simply use `DataTestUtility.ConnectionStrings`.

Reply:
Doh, done.